recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "tardis-sn" %}
+{% set version = "3.0.a3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: https://github.com/tardis-sn/tardis/archive/v3.0-alpha.3.tar.gz
+    sha256: a6bea10ebdc2b10e610953c30d6920bd914bf877f3c4418ed1d81c4d2e914289
+  - url: https://github.com/astropy/astropy-helpers/archive/v3.1.1.tar.gz
+    sha256: 8ba7e693f1cc9a1c12be3b4ba9e65b67246bff1741c726b79a6dc7b233ba4cd4
+    folder: astropy_helpers
+
+build:
+  number: 1
+  skip: True   # [win or py != 36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+  host:
+    - python
+    - pip
+    - setuptools
+    - astropy =3
+    - Cython =0.29
```

Comment:
Should be lowercase.  Also, is this really that strict or is it `>=0.29`?

Reply:
for now we are really strict - as then we know that it works.